### ISSUE TITLE
When exporting all public keys, public keys from key pairs aren't included

### ISSUE THREAD

**Issue by maniqui** [mailvelope/mailvelope#567]
On v2.2.0, when exporting all public keys from _Key management > Display keys_, the public keys for owned key pairs aren't being included in the exported file.

On v2.1.0 and below, Mailvelope would include all public keys on the exported file. 

Is this a bug or a feature?
Personally, I consider it a bug, as now I don't have a way to export _all public keys_ (that is, including my own public keys) in one step.

**Comment by toberndo** [comment]
This is intended, you should only export the keys you see in the list after you have applied a filter.

> Personally, I consider it a bug, as now I don't have a way to export all public keys (that is, including my own public keys) in one step.

You can export all public keys if you don't apply a filter and in the export dialog choose Public.

**Comment by maniqui** [comment]
My bad!
Choosing "Public" on the export dialog is how I used to do it, but I swear that yesterday, when I clicked on the "Export" button (with _Key type_ set to "All", the following UI was missing:

![image](https://user-images.githubusercontent.com/333695/37783514-70acd828-2dd4-11e8-9778-5523e76b2847.png)

Of course, the UI is there, so I might have been a hiccup on my side. Sorry. Closing the issue.
